src/components/Pages/OfficialDocuments/_Pagination.scss

```diff
@@ -98,15 +107,15 @@
   fill: #FFFFFF;
 }
 
-.coa-OfficialDocumentPage_page .coa-CheveronLeftBlue:hover{
+.coa-OfficialDocumentPage_page .coa-CheveronLeftBlue:hover {
```

Comment:
it'd be really cool if these were only "active" when there was actually a next/previous page.

right now it still "feels" clickable even though it does nothing

Reply:
Agree. I'll make on note on the issue about this moving forward.